### DISCUSSION TITLE
What happened to gtk ruby documentation?

### DISCUSSION THREAD

**Discussion by TexDh** [ruby-gnome/ruby-gnome#1470] [Q&A] [answered]
Hi, 
There is no documentation available for GTK anymore on http://ruby-gnome2.osdn.jp/.
Could you share your thoughts..why so?

**Comment by kou** [comment, marked as answer]
See https://github.com/ruby-gnome/ruby-gnome/issues/1465 .

**Reply by kojix2** [reply]
:cat2:  s/1465f/1465/
[https://github.com/ruby-gnome/ruby-gnome/issues/1465](https://github.com/ruby-gnome/ruby-gnome/issues/1465)

**Reply by kou** [reply]
Thanks. I've fixed.

**Reply by TexDh** [reply]
Many thanks for the fix.

**Comment by kojix2** [comment]
Maybe we should replace the links and put up a banner that says the previous page is gone?

**Reply by kojix2** [reply]
https://github.com/kojix2/ruby-gnome.github.io/tree/patch-3

Concept Art:  "diff" banner

**Reply by kou** [reply]
I set redirect to https://ruby-gnome.github.io/ from http://ruby-gnome2.osdn.jp/ .

**Reply by TexDh** [reply]
Indeed this will at least prevent the new issues/discussions opening.
Thank you!
